MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java

```diff
@@ -38,7 +38,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // Get the location engine object for later use.
     LocationEngine locationEngine = LocationSource.getLocationEngine(this);
-    locationEngine.activate();
     locationEngine.requestLocationUpdates();
```

Comment:
For some reason, the example breaks when requestLocationUpdates() is removed. That's why I left it in there.